metricbeat/docs/developer-guide.asciidoc

```diff
@@ -1,10 +1,212 @@
-= Add a module
+= Metricbeat Contributions
 
-* Basic files
-* Configuration
-* Define MetricSets in advance
+Metricbeat can be used in two ways:
```

Comment:
I would add here that "Metricbeat is periodically interrogating an external system to fetch key metrics information" to say from the beginning what type of systems can be added to Metricbeat.